README.md

```diff
@@ -45,7 +47,7 @@ Then add the following to your **module's** build.gradle:
 ```gradle
 dependencies {
     ...
-    implementation 'com.raygun:raygun4android:4.+'
+    implementation 'com.raygun:raygun4android:4.0.1'
```

Comment:
What's the reason behind this change?

Reply:
I want to tell people that at this stage - until anything else is published - 4.0.1 is the stable released. `4.+` might just unintendend suck in a 4.1.0 alpha/snapshot when I publish it.